src/components/ContestDetails.tsx

```diff
@@ -37,6 +38,8 @@ export type Vote = {
 }
 
 type ContestDetailsState = {
+  showViewer: boolean
```

Comment:
In the spirit of what we looked over yesterday, you probably dont need this state since you can just use the value of activeViewerIndex for that

Regardless, looks good to me!